content/project/project_management.md

```diff
@@ -0,0 +1,196 @@
+* *Zenhub:* Project management layer on top of Github, for reduced friction between development and project management. Scales to complex projects better than native Github features, but is still pushed to the limits by our larger projects.
+
+### Communications
+
+Projects need a platform for internal communications between team members, as well as range of different channels for communicating with different stakeholers (see [Project Community](../project_community/#community-engagement/)). Remember, projects thrive on comprehensive communication. It is vital that projects not only have the ability to reach out to peers and close stakeholders, but that team members feel comfortable enough with the tools to do so. Whatever platform you choose, it should
+* Facilitate asynchronous communication
+* Keep written records of staff communication
+* Organize communication records for easier recovery
+
+Communication platforms are usually chosen based on cultural norms -- again, it's important that team members feel comfortable communicating on the platform. OMSF project primarily use Slack, with some common norms:
+* Public channels for engaging with stakeholders
+* Private channels for high-volume staff communication
+* One Channel for each subproject / deliverable
+* One Channel for each group of stakeholders
+* One Channel for each software product
+* Use group DMs sparingly - named channels allow more flexibility
+
+OMSF projects also use Gmail for private, formal correspondence, including some engagement with industry partners.
+
+## Evaluation and metrics
+
+Simply put, metrics are measurable changes over time. More often than not, these are quantitative measurements that represent the state of a project. Metrics are almost never useful as individual points - they require a baseline and some time to determine health. Measuring Open Source Software is a hard problem, and OMSF projects have not settled on any specific set of metrics to track. We have considered
+* *Github statistics:* These primarily reflect the actions of the developer team, except in very widely used, mature projects.
+* *Package downloads:* Difficult to distinguish users from automated downloads in Continuous Integration pipelines.
+* *Citations:* Nonexistent for young projects. Difficult to normalize for mature projects, as software tools may be described in a number of publications, making it unclear which papers' citations should be included.
+
+## Publishing scientific papers
+
+If your software project is successful, it will lead to new, publication-worthy science. Publishing a scientific paper is a much different kind of task than software development, and requires different expertise and considerations. The following is adapted from [Open Force Field's writing process and stages](https://docs.google.com/document/d/1tvkGiP5N_T2v7iH_Y1R69V-pXJUnTS8RqEnwf_TG-wM/edit?usp=sharing)
```

Comment:
```suggestion
If your software project is successful, it will lead to new, publication-worthy science. Publishing a scientific paper is a much different kind of task than software development, and requires different expertise and considerations. The following is adapted from [Open Force Field's writing process and stages](https://docs.google.com/document/d/1tvkGiP5N_T2v7iH_Y1R69V-pXJUnTS8RqEnwf_TG-wM/edit?usp=sharing).
```

period at end of paragraph